### ISSUE TITLE
Consider fixing prose of `/vuln:Vuln.../Ack...s/Ack.../Name` from CVRF v1.2 to CSAF v2.0

### ISSUE THREAD

**Issue by sthagen** [oasis-tcs/csaf#101]
As @tschmidtb51 pointed out, the production [6.15.1.1](http://docs.oasis-open.org/csaf/csaf-cvrf/v1.2/cs01/csaf-cvrf-v1.2-cs01.html#_Toc493508958) in CVRF v1.2 is contradicting the parent element description of the cardinality in section [6.15.1](http://docs.oasis-open.org/csaf/csaf-cvrf/v1.2/cs01/csaf-cvrf-v1.2-cs01.html#_Toc493508957) and in any case not with the [XSD schema](http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln).

```
6.15.1.1 Vulnerability – Acknowledgements – Acknowledgement – Name

Element vuln:Name
« The vuln:Name MUST be present with cardinality [1, ∞] in vuln:Acknowledgment and every instance given contains the name of the party being acknowledged. » [CSAF-6.15.1.1-1]
```

should read:

```
6.15.1.1 Vulnerability – Acknowledgements – Acknowledgement – Name

Element vuln:Name
« The vuln:Name MUST be present with cardinality [0, ∞] in vuln:Acknowledgment and every instance given contains the name of the party being acknowledged. » [CSAF-6.15.1.1-1]
```

From a perspective 3 years later these rules on containers forced as `[1, ∞]` allowing all children to be `[0, ∞]` reads weird.
But then, this is XSD/XML. 
In JSON we tend to allow removal of empty containers, or even expect that. So, the real fix would be to make this explicit and allow the parent container `Acknowledgement` to be optional i.e. to be of cardinality `[0, ∞]` and not `[1, ∞]`.

I think what we should communicate is, that of all nested containers, the tree should be shaken, such that any container (object or array) that is spreading from root has at least one descendant leaf element which is not empty (or `null`).

**Comment by tschmidtb51** [comment]
I agree that the tree should be shaken. This makes it much easier to read. We also prepare that in the current schema by #81, #83 and #86.
